src/wp-includes/theme.php

```diff
@@ -2989,6 +2989,150 @@ function require_if_theme_supports( $feature, $include ) {
 	return false;
 }
 
+/**
+ * Registers a theme feature for use in {@see add_theme_support}.
+ *
+ * This does indicate that the current theme supports the feature, it only describes the feature's contents.
```

Comment:
Is this missing a `not` here?

```suggestion
 * This does not indicate that the current theme supports the feature, it only describes the feature's contents.
```

Reply:
Yep, thanks for the catch!